packages/terra-core-docs/src/terra-dev-site/doc/icon/AccessabilityGuide.2.doc.mdx

```diff
@@ -0,0 +1,111 @@
+
+<Whitespace />
+
+## Accessibility Considerations:
+
+### Meaningful Icons
+
+#### Informative Icons
+
+Icons that convey information **must have alternative text in code** to ensure that assistive technologies pass the icon's meaning to the user.
+
+<InformativeIconExamples/>
+
+In this example the factory icon (<IconManufacturer/>) and announcement icon (<IconAnnouncement/>) are both informative. The factory icon has the meaning that _this text is an address_. The announcement icon has the meaning that _this text is an announcement_. However, neither icon conveys that meaning to users who cannot see them. Therefore, both icons code must include `title="<alternative text>"` to adequately convey their meaning to a screen reader user. The factory icon should have `title="Manufactured by"` and the announcement icon should have `title="Announcement"`. With these text alturnatives added, screen reader users will hear _"&lt;Text Alternative&gt; &lt;Text&gt_":
+
+<p><em>"Manufactured by 2800 Rock Creek Pkwy Kansas City, MO 64117"</em></p>
+<p><em>"Announcement Facilities will closed on Monday"</em></p>
+
+#### Functional Icons
+
+Icons that perform some operation, like an icon taht is part of a hyperlink or a button, are said to be functional icons. Functional icons, like informative icons, **must have alternative text in code** to ensure that assistive technologies pass the icon's meaning to the user.
+
+<FunctionalIconExample/>
+
+In the example above, links to view a folder in three different ways are given: as a list (<IconList/>), as tiles (<IconTile/>), and as a tree map (<IconTreemap/>). With the title provided to each icon, the user will hear:
+
+<p><em>"View folder as, link, List view, link, Tile view, link, Treemap view"</em></p>
+
+#### Groups of Icons
+
+Sometimes icons only have meaning in the context of other icons. Here is one example where multiple icons are used to create a single five-star rating system:
+
+<GroupOfIconsExample/>
+
+One technique to make groups of images accessible is show above - to give the first image in the group a meaning but mark the other images as presentationsl (so that they will be ingored by screen readers). This technique results in the user hearing:
```

Comment:
```suggestion
One technique to make groups of images accessible is shown above - to give the first image in the group a meaning but mark the other images as presentational (so that they will be ignored by screen readers). This technique results in the user hearing:
```